app/controllers/duties_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class DutiesController < ApplicationController
+  def index
+    @duties = Duty.all
+    tr = TimeRange.all.sort_by(&:start_time)
```

Comment:
`TimeRange.order(:start_time)`

`.all.sort_by` loads everything into Rails's memory and sorts them in place, while `.order` calls the `ORDER BY` SQL command and takes advantage of the database's representation.